docs/src/app/components/CodeExample/CodeBlockTitle.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import IconButton from 'material-ui/lib/icon-button';
+import CodeIcon from 'material-ui/lib/svg-icons/action/code';
+import Toolbar from 'material-ui/lib/toolbar/toolbar';
+import ToolbarGroup from 'material-ui/lib/toolbar/toolbar-group';
+import ToolbarTitle from 'material-ui/lib/toolbar/toolbar-title';
+
+const CodeBlockTitle = (props) => (
+  <Toolbar>
+    <ToolbarGroup float="left">
+      <ToolbarTitle text={props.title || 'Example'} />
+    </ToolbarGroup>
+    <ToolbarGroup float="right">
+      <IconButton touch={true}>
```

Comment:
Looking at the current tooltip code, and it's utilisation, I'd rather not. I also think it would be visually distracting.

And honestly, if someone can't figure out how to show the code given the current visual clues (pointer change, < > icon), they're probably not going to get very much further. :smirk:

If it becomes a problem, we can revisit.